### PR TITLE
Add rubygem-ovirt-engine-sdk package

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Preparation for https://github.com/fog/fog-ovirt/pull/10

### CODE REVIEW

**Comment by iNecas** [comment]
FYI @orrabin

File: packages/foreman/rubygem-ovirt-engine-sdk/rubygem-ovirt-engine-sdk.spec [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+%{gem_libdir}
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.adoc
+%doc %{gem_instdir}/CHANGES.adoc
+
+%changelog
```
Comment (ekohl) [inline packages/foreman/rubygem-ovirt-engine-sdk/rubygem-ovirt-engine-sdk.spec]: In nightly we need a changelog entry.

Comment (iNecas) [reply packages/foreman/rubygem-ovirt-engine-sdk/rubygem-ovirt-engine-sdk.spec]: Updated

**Comment by iNecas** [comment]
BTW I was also checking in production and the scratch rpm was installed properly and I was able to do `require 'ovirtsdk4'`

**Comment by ekohl** [comment]
Thanks! Was waiting for the build to complete.

**Pull request merged**